docs/guides/error-mitigation-and-suppression-techniques.ipynb

```diff
@@ -207,11 +208,98 @@
     "estimator.options.resilience.zne.extrapolator = \"exponential\""
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "e5dcab7b",
+   "metadata": {},
+   "source": [
+    "## Probabilistic Error Amplification (PEA)\n",
+    "\n",
+    "One of the main challenges in ZNE is to accurately amplify the noise affecting the target circuit. Gate folding provides an easy way to perform this amplification, but is potentially inaccurate and might lead to incorrect results.  See the article [\"Scalable error mitigation for noisy quantum circuits produces competitive expectation values\"](https://www.nature.com/articles/s41567-022-01914-3), and specifically page 4 of the [supplementary material](https://arxiv.org/pdf/2108.09197) for details. \n",
+    "\n",
+    "PEA is a more sophisticated technique that performs preliminary experiments to reconstruct the noise and then uses this information to perform an accurate amplification. For utility-scale experiments, it is often the best choice.\n",
+    "\n",
+    "Because PEA is a ZNE amplification technique, you also need to enable ZNE by setting `resilience.zne_mitigation = True`. Other `resilience.zne` options can be used in addition to set extrapolators, amplification levels, and so on. PEA requires a noise model, which is automatically generated when using primitives. \n",
```

Comment:
```suggestion
    "Because PEA is a ZNE noise amplification technique, you also need to enable ZNE by setting `resilience.zne_mitigation = True`. Other `resilience.zne` options can be used in addition to set extrapolators, amplification levels, and so on. PEA requires a noise model, which is automatically generated when using primitives. \n",
```